WootricSDK/WootricSDK/WTRSurveyViewController.m

```diff
@@ -36,6 +36,7 @@
 #import "Wootric.h"
 #import "WTRSurveyDelegate.h"
 #import <Social/Social.h>
+#import <WootricSDK/WootricSDK-Swift.h>
```

Comment:
Why do we need this here?

Reply:
When we are using swift code in Objective C then we have to import swift header file like above.